tests/README.md

```diff
@@ -242,6 +242,8 @@ azure:
     #image_region: eu-central-1
     # already existing image in Azure to be used for testing (required/alternatively optional)
     image_name: <image name>
+    # hyperV generation to use. Possible values are 'V1' and 'V2'. Default is 'V1'
+    hyper_v_generation: V2
```

Comment:
According to the default in `azure.py` we might set this to `V1`, too. This could avoid confusions while already mentioning the that `V1` and `V2` are accepted values.